src/explore-education-statistics-common/src/modules/table-tool/types/selectedPublication.ts

```diff
@@ -15,4 +17,5 @@ export interface SelectedRelease {
   slug: string;
   latestData: boolean;
   title: string;
+  type: ReleaseType;
```

Comment:
This is the short-lived type extension I mentioned above. I'd loathe to add this here, but conscious of scope creep if we do EES-4312 in this PR. Happy to go either way, just wanna flag it :)

Reply:
It's fine, eventually we'll clean this up :smile: